src/datafactory/azext_datafactory/generated/_help.py

```diff
@@ -19,28 +19,28 @@
 
 helps['datafactory factory list'] = """
     type: command
-    short-summary: Lists factories under the specified subscription.
+    short-summary: "Lists factories under the specified subscription."
     examples:
       - name: Factories_ListByResourceGroup
         text: |-
-               az datafactory factory list --resource-group "myResourceGroup"
+               az datafactory factory list --resource-group "exampleResourceGroup"
```

Comment:
The convention is to use myResourceGroup. What caused this change?

Reply:
The previous one is generated from my local changed swagger example name. I remember myResourceGroup is only needed in the generated test? If it's mandatory. we can change it in codegen.